docs/architecture/architecture.tex

```diff
@@ -80,16 +87,24 @@ \subsection{Subsystem decomposition}
 	\label{fig:modules}
 \end{figure}
 
-The input module should be able to read the data. With that data a dataModel
-can be constructed. \\
-The Analysis Specification module processes the script that defines how the
-data should be analyzed. The user provides this script. This module translate
-the script in operations that can be performed by the Analysis Module.\\
-The Analysis Module perform the analyses over the dataModel. The output of this
-module is the result of the analysis, this is a dataModel.\\
-The Visualization module creates a certain visual representation of a
-dataModel. For example it can create a box-plot of the creatine levels.
+\begin{description}
+\item[Input]
+The input module is able to read the data from provided files that are specified in an xml file. With that data a dataModel can be constructed.
+\item[Model]
+The model module contains and manages the data that was read and/or analyzed by the user. This data will also be provided to the controller and analysis modules. The data is stored in tables.
+\item[Analysis Specification]
+The Analysis Specification module processes the SUGAR script that defines how the data should be analyzed. The user provides this script. In this module the script is parsed in operations that can be performed by the Analysis Module.
+\item[Analysis]
```

Comment:
misschien noemen dat momenteel de analyse module qua packagestructuur in de model zit, maar dat dat komende week veranderd moet worden.